site/topics/logic/boolean-logic.rst

```diff
@@ -90,6 +90,248 @@ Boolean Operators and Operands
 Truth Tables
 ============
 
+* Truth tables provide a structured visualization of all possible truth values for logical expressions
+* These are probably best understood with examples
+* Below is a truth table for the above boolean operators for all possible combinations of values for two operands
+
+
+.. list-table:: Truth Table for Basic and Common Logical Operators
+    :widths: auto
+    :align: center
+    :header-rows: 1
+
+    * - :math:`a`
+      - :math:`b`
+      -
+      - :math:`\lnot a`
+      - :math:`a \land b`
+      - :math:`a \lor b`
+      -
+      - :math:`a \oplus b`
+      - :math:`\lnot (a \land b)`
+      - :math:`\lnot (a \lor b)`
+    * - :math:`false`
+      - :math:`false`
+      -
+      - :math:`true`
+      - :math:`false`
+      - :math:`false`
+      -
+      - :math:`false`
+      - :math:`true`
+      - :math:`true`
+    * - :math:`false`
+      - :math:`true`
+      -
+      - :math:`true`
+      - :math:`false`
+      - :math:`true`
+      -
+      - :math:`true`
+      - :math:`true`
+      - :math:`false`
+    * - :math:`true`
+      - :math:`false`
+      -
+      - :math:`false`
+      - :math:`false`
+      - :math:`true`
+      -
+      - :math:`true`
+      - :math:`true`
+      - :math:`false`
+    * - :math:`true`
+      - :math:`true`
+      -
+      - :math:`false`
+      - :math:`true`
+      - :math:`true`
+      -
+      - :math:`false`
+      - :math:`false`
+      - :math:`false`
+
+
+
+* In the context of digital circuits, it is common to use ``0`` and ``1`` in place of :math:`true`` and :math:`false``
+
+.. list-table:: Truth Table for Basic and Common Logical Operators using ``0`` and ``1``
```

Comment:
Repeating the table feels unnecessary. I'd hope students can replace a symbol with a different symbol in their head by now.

Reply:
Good call --- removed